Jenkinsfile

```diff
@@ -0,0 +1,85 @@
+pipeline {
+    options {
+        disableConcurrentBuilds()
+        timestamps()
+    }
+    agent {
+        node { label 'master' }
+    }
+    environment {
+        GITHUB_CREDENTIALS = credentials('brave-builds-github-token-for-pr-builder')
+        BC_REPO = "https://${GITHUB_CREDENTIALS}@github.com/brave/brave-browser"
+        BC_BRANCH = "brave-core-${GIT_BRANCH}"
+    }
+    stages {
+        stage('checkout') {
+            steps {
+                echo "BRANCH_NAME: ${BRANCH_NAME}"
+                echo "CHANGE_TARGET: ${CHANGE_TARGET}"
+                echo "CHANGE_ID: ${CHANGE_ID}"
+                echo "CHANGE_TITLE: ${CHANGE_TITLE}"
+                echo "CHANGE_AUTHOR: ${CHANGE_AUTHOR}"
+                echo "CHANGE_AUTHOR_DISPLAY_NAME: ${CHANGE_AUTHOR_DISPLAY_NAME}"
+                sh """
+                    if [ -d brave-browser ]; then
+                        rm -rf brave-browser
+                    fi
+
+                    git clone -b ${CHANGE_TARGET} ${BC_REPO}
+
+                    cd brave-browser
+
+                    if [ -z "`git ls-remote --heads ${BC_REPO} ${BC_BRANCH}`" ]; then
+                        git checkout -f -b ${BC_BRANCH}
+                    else
+                        git checkout -f ${BC_BRANCH}
+                    fi
+                """
+            }
+        }
+        stage('config') {
+            steps {
+                sh """
+                    git config -f brave-browser/.git/config user.name brave-builds
+                    git config -f brave-browser/.git/config user.email devops@brave.com
+                    git config -f brave-browser/.git/config push.default simple
```

Comment:
Nice! Thanks for pulling this in (just ran into that stupid warning myself, as you saw 😄)